deployment/config.go

```diff
@@ -113,6 +113,8 @@ type TerraformDBSettings struct {
 	EnablePerformanceInsights bool `default:"false"`
 	// A list of DB specific parameters to use for the created instance.
 	DBParameters DBParameters
+	// ClusterIdentifier indicates to point to an existing cluster
+	ClusterIdentifier string `default:""`
```

Comment:
To understand, would any of the above settings be applicable when `ClusterIdentifier` is used?

Reply:
Yep, it would. All of them should be applicable I believe.